src/nvcgo/Makefile

```diff
@@ -31,8 +31,8 @@ OBJ_NAME := $(LIB_NAME).so
 HDR_NAME := $(LIB_NAME).h
 CTYPES_H := ctypes.h
 
-CGO_CFLAGS  := -std=gnu11 -O2
-CGO_LDFLAGS := -Wl,--gc-sections -Wl,-s -Wl,-soname,$(LIB_SONAME)
+CGO_CFLAGS  := -std=gnu11 -O2 $(CGO_CFLAGS)
```

Comment:
Does using `+=` over `:=` make sense in this context?

Reply:
Using `+=` instead of `:=` would change the order of the flags. The user's `CGO_CFLAGS` should be at the end so they can override the project defaults (particularly the optimization level).